rollup.config.js

```diff
@@ -55,21 +52,13 @@ export default [
   // NPM bundles. They have all the dependencies excluded for end code size optimization.
   {
     ...commonInput,
-    external: Object.keys(dependencies),
     output: [
       // CJS for usage with `require()`
       {
         ...commonOutput,
         file: `${outputDirectory}/${artifactName}.cjs.js`,
         format: 'cjs',
       },
-
-      // ESM for usage with `import`
-      {
-        ...commonOutput,
-        file: `${outputDirectory}/${artifactName}.esm.js`,
-        format: 'es',
-      },
```

Comment:
As far as I understand this package is intended to be used only via Open CDN. Open CDN supports only the ESM format. So you must leave the ESM format and may remove the other output formats. The less the NPM package size, the faster Open CDN will load it.

Reply:
The IIFE outputs should be removed too. Open CDN can produce IIFE from ESM if you need IIFE.